src/tables/ctran_data.py

```diff
@@ -78,12 +78,13 @@ def __init__(self, user=None, passwd=None, hostname=None, db_name=None, schema="
 
     # [dev tool]
     # This will create a mock CTran Table for development purposes.
-    def create_table(self, ctran_sample_path="assets/"):
+    # Updated function so that sample name can be passed: used for end-to-end testing, where separate test data needs to be loaded
+    def create_table(self, ctran_sample_path="assets/", ctran_sample_name="/ctran_trips_sample.csv", exists_action="append"):
         if not isinstance(self._engine, Engine):
             self._print("ERROR: self._engine is not an Engine, cannot continue.")
             return False
 
-        csv_location = "".join([ctran_sample_path, "/ctran_trips_sample.csv"])
+        csv_location = "".join([ctran_sample_path, ctran_sample_name])
         self._print("Loading " + csv_location)
```

Comment:
I disagree with the change of `"/ctran_trips_sample.csv"` to `"ctran_trips_sample.csv"`, Python will not throw an error when `assets//ctran_trips_sample.csv` is used and it allows for forgiveness of the user if they don't add a `/` to the end of ctran_sample_path should they change it.

Reply:
I don't think I changed it, I just moved it to function parameters (to be able to import test csv)

Reply:
Oh you did, I can't read. My bad.

Reply:
All good, thanks for checking)